packages/agent/test/services/model-customizations/customization.test.ts

```diff
@@ -0,0 +1,172 @@
+        ).mockResolvedValue([webhookAction, updateRecordAction]);
+
+        const customizer = dataSourceCustomizerFactory.mockAllMethods().build();
+        const collection = {
+          addAction: jest.fn().mockReturnThis(),
+        };
+        (customizer.getCollection as jest.Mock).mockReturnValue(collection);
+
+        await actionCustomization.addCustomizations(
+          customizer,
+          undefined as unknown as CollectionCustomizer,
+        );
+
+        expect(collection.addAction).toHaveBeenCalledWith('myWebhookAction', expect.anything());
+      });
+    });
+
+    describe('when experimental.updateRecordCustomActions is define', () => {
+      it('should add all update record action customizations', async () => {
+        const options = forestAdminHttpDriverOptions.build({
+          experimental: {
+            updateRecordCustomActions: true,
+          },
+        });
+
+        const actionCustomization = new CustomizationPluginService(options);
+
+        (
+          options.forestAdminClient.modelCustomizationService.getConfiguration as jest.Mock
+        ).mockResolvedValue([webhookAction, updateRecordAction]);
+
+        const customizer = dataSourceCustomizerFactory.mockAllMethods().build();
+        const collection = {
+          addAction: jest.fn(),
+        };
+        (customizer.getCollection as jest.Mock).mockReturnValue(collection);
+
+        await actionCustomization.addCustomizations(
+          customizer,
+          undefined as unknown as CollectionCustomizer,
+        );
+
+        expect(collection.addAction).toHaveBeenCalledWith(
+          'myUpdateRecordAction',
+          expect.anything(),
+        );
+      });
+    });
+
+    it.each([false, undefined])(
+      'should not add customizations when the feature is disabled (using %s)',
+      async optionValue => {
+        const options = forestAdminHttpDriverOptions.build({
+          experimental: {
+            webhookCustomActions: optionValue,
+            updateRecordCustomActions: optionValue,
+          },
+        });
+
+        const actionCustomization = new CustomizationPluginService(options);
+
+        (
+          options.forestAdminClient.modelCustomizationService.getConfiguration as jest.Mock
+        ).mockResolvedValue([webhookAction, updateRecordAction]);
+
+        const customizer = dataSourceCustomizerFactory.mockAllMethods().build();
+        const collection = {
+          addAction: jest.fn(),
+        };
+        (customizer.getCollection as jest.Mock).mockReturnValue(collection);
+
+        await actionCustomization.addCustomizations(
+          customizer,
+          undefined as unknown as CollectionCustomizer,
+        );
+
+        expect(collection.addAction).not.toHaveBeenCalled();
+      },
+    );
+  });
+
+  describe('buildFeatures', () => {
+    it(
+      'should return all feature activated (using options.experimental) ' +
```

Comment:
```suggestion
      'should return all activated features (using options.experimental) ' +
```

Was the new line intentional ?